primitives/rpc/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "sp-rpc"
-version = "6.0.0"
+version = "7.0.0"
```

Comment:
Why?

Reply:
`sc-tracing` is using sp-rpc v7.

Reply:
How does that work on master? Something seems to be wrong here or something maybe pulls the dep now from crates.io?

Reply:
As we don't need the RPC crate we don't need to bump this version as we're not pulling in `sc-tracing` any more so this resolves.